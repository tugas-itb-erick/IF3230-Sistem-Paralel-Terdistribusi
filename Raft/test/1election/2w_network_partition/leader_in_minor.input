5
set_timeout 5 0
simulate 10
disconnect 1 3
disconnect 1 4
disconnect 1 5
disconnect 2 3
disconnect 2 4
disconnect 2 5
simulate 100
connect 1 3
connect 1 4
connect 1 5
connect 2 3
connect 2 4
connect 2 5
set_timeout 1 0
timestep
get_role 1
flush_outbox
timestep
flush_outbox
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
end_simulation