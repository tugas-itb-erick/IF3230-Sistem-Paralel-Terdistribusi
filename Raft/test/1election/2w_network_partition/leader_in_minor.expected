candidate
leader
follower
follower
follower
follower
